package.json

```diff
@@ -45,15 +47,24 @@
   },
   "dependencies": {
     "@azure/cosmos": "^3.7.2",
+    "@babel/runtime": "^7.9.0",
```

Comment:
why do we need this dep?

Reply:
Because gulp use a babel/core function for generating a template. Without this deep we got this:
```
internal/modules/cjs/loader.js:583
    throw err;
    ^

Error: Cannot find module '@babel/runtime/helpers/createSuper'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/Users/alessio/Public/Workspace/io-functions-commons/node_modules/mjml-core/lib/index.js:66:44)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
```

Reply:
these are all devDependencies anyway